src/ros_commo.py

```diff
@@ -444,10 +445,8 @@ def __init__(self):
 			self.language_perceived_text_cb)
 
 		# Chat infrastructure text.
-		# from chatbot.msg import ChatMessage
-		#rospy.Subscriber("/robot/speech", chatbot/ChatMessage,
-		#rospy.Subscriber("chatbot_speech", chatbot/ChatMessage,
-		#	self.chat_perceived_text_cb)
+		rospy.Subscriber("chatbot_speech", ChatMessage,
```

Comment:
The ChatMessage is a part of the hanson robotics software stack.  It is not available as open source.  This means the code will break for non-HR employees.

Even if it was publicly available, it adds a new dependency on a module that would not otherwise be required -- forcing users to install yet more subsystems that are not actually useful.

The biggest problem with the ChatMessage is that it is a string and a floating point number, but that floatingpoint number is not used anywhere.  Its not set, its not needed.  It was a design flaw to create/use ChatMessage.